logstash-core/src/main/java/org/logstash/plugins/aliases/AliasPlugin.java

```diff
@@ -11,6 +11,11 @@
  */
 public class AliasPlugin {
 
+    public AliasPlugin(String aliasName, String from, List<AliasDocumentReplace> docHeaderReplaces) {
+        this.aliasName = aliasName;
+        this.from = from;
+        this.docHeaderReplaces = docHeaderReplaces;
+    }
```

Comment:
Constructor should be moved after fields definitions.
Also remove the `@JsonProperty` annotations form fields because not read anymore through Jackson